angular/src/app/app.component.ts

```diff
@@ -0,0 +1,226 @@
+import { Component, Renderer2, NgZone } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { environment } from '../environments/environment.development';
+
+declare const Helpshift: any;
+declare const helpshiftConfig: any;
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.css',
+})
+export class AppComponent {
+  title = 'Web Chat';
+
+  constructor(private renderer: Renderer2, private ngZone: NgZone) {}
+
+  isLoggedIn: boolean = false;
+  selectedPrivacyOption: string = 'disable';
+  selectedLauncherOption: string = 'showLauncher';
+  position: string = 'bottom-right';
+  selectedFullScreenOption: string = 'exitFullScreen';
+  unreadMessageCount: any = 0;
```

Comment:
Can we have the type as [Number](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean) ?

Reply:
Yes will do this